src/Api.Common/Domain/Token.cs

```
@@ -18,10 +18,17 @@ public class VaultToken
         {
             var properties = new Dictionary<string, string>
             {
-                {VaultTokenClaims.ApiKeyId, vaultApiKey.ToString()},
+                {VaultTokenClaims.ApiKeyId, vaultApiKey.Id.ToString()},
                 {VaultTokenClaims.TenantId, vault.TenantId},
                 {VaultTokenClaims.UniqueName, vaultApiKey.Name},
                 {VaultTokenClaims.VaultId, vault.Id.ToString()},
+                {
+                    VaultTokenClaims.VaultType, vault.Type switch{
+                        VaultType.Private => "private",
+                        VaultType.Shared => "shared",
```

we'll never get the `shared` token via API. Should throw an exception here